tomas/index.js

```diff
@@ -0,0 +1,17 @@
+// NaN = not a number
+
+// Check if "hello" * 1 is NaN
+let str = 'hello'
+console.log('"hello" * 1 is NaN -->', isNaN(str * 1))
+
+// Check if 100em is NaN
+let valueToCheck = '100em'
+console.log('100em not a number -->', isNaN(valueToCheck))
+
+// What is the type of NaN?
+console.log('What is the type of NaN? -->', typeof NaN)
+
+// rounding
+
+let mult = .1 * .2
+console.log('rounded sum -->', mult.toFixed(2))
```

Comment:
You forgot to solve task3!
console.log((12.2600000000033).toFixed(2));
console.log((0.6378399).toFixed(1));
console.log((1.84222).toFixed(4));